addon/webextension/selector/callBackground.js

```diff
@@ -4,10 +4,10 @@
 
 this.callBackground = function callBackground(funcName, ...args) {
   return browser.runtime.sendMessage({funcName, args}).then((result) => {
-    if (result.type === "success") {
+    if (result && result.type === "success") {
       return result.value;
-    } else if (result.type === "error") {
-      let exc = new Error(result.message);
+    } else if (result && result.type === "error") {
+      let exc = new Error(result.message || "Unknown error");
       exc.name = "BackgroundError";
       if ('errorCode' in result) {
```

Comment:
I think you'll need to guard against `result` being undefined in these `"foo" in result` clauses, too. They'll throw if result is falsy.